index.js

```diff
@@ -842,7 +850,8 @@ const rpcMethods = {
             }
         }
         if (ps.TYPE === 'ENUM') {
-            payload.hm.ENUM = ps.VALUE_LIST[params[3]];
+            payload.val = ps.VALUE_LIST[params[3]];
```

Comment:
this is a change that would break a few things in my scripts and in my ui... hmmm... :)

Reply:
Thought so, how about:

`--protocol-option-replace-colons` for [this](https://github.com/dersimn/hm2mqtt.js/blob/8c07bd8586afb34b040982aba68d56b3c8735e70/config.js#L21)
`--protocol-option-direct-enums`